setup.py

```diff
@@ -34,8 +34,7 @@ def setup_options(name, version):
             "xmltodict>=0.9.1",
             "attrs>=15.1.0",
             "testtools>=1.7.1,<1.8.0",
-            "iso8601>=0.1.10",
-            "toolz>=0.7.4"
+            "iso8601>=0.1.10"
```

Comment:
For future reference, the convention in Python (I am surprised to see we weren't doing it here!) is to always put a trailing comma at the end of the line to avoid spurious diffs like this.